packages/formStructure/README.md

```diff
@@ -5,26 +5,35 @@ These guidlines and components help us maintain consistent spacing in our forms,
 ## Components
```

Comment:
where is this readme used?

Reply:
In the "Form structure" Storybook stories. For example: https://dcos-labs.github.io/ui-kit/?path=/story/forms-form-layout--formtitle

Reply:
OK